### PR TITLE
Fix login page styling

### LINKED ISSUE #4795: Login Page styling regressed
### Describe the bug

The login page styling looks weird. The gray boxes look bad and they are new. See this space: https://huggingface.co/spaces/gradio/hello_login

image

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

Go to this space https://huggingface.co/spaces/gradio/hello_login

Seems to be a regression introduced in the last release

### Screenshot

_No response_

### Logs

_No response_

### System Info

```shell
gradio 3.35.2
```

### Severity

I can work around it

### PR DESCRIPTION
**Pull request by freddyaboulton**
## Description

With #4374 , `container` is set to false by default when it should be true (matches the default value in the python api). 

~Working on storybook for this.~ PR is ready for review. Storybook is working but the font is off. We can fix storybook later because I don't want to hold up release.

Fixes #4795

## 🎯 PRs Should Target Issues

Before your create a PR, please check to see if there is [an existing issue](https://github.com/gradio-app/gradio/issues) for this change. If not, please create an issue before you create this PR, unless the fix is very small. 

Not adhering to this guideline will result in the PR being closed. 

## Tests & Changelog

1. PRs will only be merged if tests pass on CI. To run the tests locally, please set up [your Gradio environment locally](https://github.com/gradio-app/gradio/blob/main/CONTRIBUTING.md) and run the tests: `bash scripts/run_all_tests.sh`

1. You may need to run the linters: `bash scripts/format_backend.sh` and `bash scripts/format_frontend.sh`
  
1. Unless the pull request is labeled with the "no-changelog-update" label by a maintainer of the repo, all pull requests must update the changelog located in `CHANGELOG.md`:

Please add a brief summary of the change to the Upcoming Release section of the `CHANGELOG.md` file and include
a link to the PR (formatted in markdown) and a link to your github profile (if you like). For example, "* Added a cool new feature by `[@myusername](link-to-your-github-profile)` in `[PR 11111](https://github.com/gradio-app/gradio/pull/11111)`".

### CODE REVIEW

File: js/app/src/Login.stories.svelte [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+			value: "Gradio Button"
+		}
+	}}
+/>
+
+<Template let:args>
+	<Login {...args} />
+</Template>
+
+<Story name="Login page without auth message" args={{ auth_message: null }} />
```
Comment (freddyaboulton) [inline js/app/src/Login.stories.svelte:23]: Was super easy to set this up! However, the theme is not being applied properly. The page is using a seriffed-font. Any idea why @hannahblair @pngwn ?

image

**Comment by abidlabs** [comment]
Thanks @freddyaboulton! Confirming that the original issue has been fixed.

One unrelated thing I noticed is that if auth is enabled, then you don't see the gradio demo inline in jupyter/colab. This is because auth used to not work in embedded environments. However, this issue is fixed and it would be good to allow people to run code with auth enabled in jupyter/colab easily.

Could we make a quick update here:

Replace:

```py
        # Check if running in a Python notebook in which case, display inline
        if inline is None:
            inline = utils.ipython_check() and (self.auth is None)
        if inline:
            if self.auth is not None:
                print(
                    "Warning: authentication is not supported inline. Please"
                    "click the link to access the interface in a new tab."
                )
```

with

```py
        # Check if running in a Python notebook in which case, display inline
        if inline is None:
            inline = utils.ipython_check() 
        if inline:
```

**Comment by freddyaboulton** [comment]
Thanks @abidlabs ! Will make the requested change to render in embedded environments when I’m back home.

**Comment by abidlabs** [comment]
LGTM @freddyaboulton! Agree we can do storybook later

**Review by abidlabs — APPROVED** [review]

File: js/app/src/Login.stories.svelte (hunk 1 above)
Comment (pngwn) [reply js/app/src/Login.stories.svelte:23]: I'm not seeing this behaviour. The font looks correct to me.

**Review by pngwn — APPROVED** [review]
Looks good to me! Thanks for taking care of this @freddyaboulton!

File: js/app/src/Login.stories.svelte (hunk 1 above)
Comment (freddyaboulton) [reply js/app/src/Login.stories.svelte:23]: It fixed itself!!

**Pull request merged**